crates/cli/src/subcommands/generate/csharp.rs

```diff
@@ -343,11 +345,14 @@ fn autogen_csharp_product_table_common(
                 field_name.to_case(Case::Pascal),
                 default_init(ctx, &field.algebraic_type)
             );
+            if i + 1 != num_fields {
+                writeln!(output);
```

Comment:
This is causing an additional space in between each field, do we need this change? This is causing the diff for the API upgrade to be pretty insane: https://github.com/clockworklabs/BitCraftClient/pull/3586/files

Reply:
Oh right, I added this to make the emitted fields more readable, as the lines can get long. It's not a needed change tho, purely style. Should we keep or remove it?

Reply:
please remove it, if you feel strongly about the style change we can do it in a separate PR IMO - I just want the API upgrade to be as simple as possible.